src/main/java/com/zenjava/javafx/maven/plugin/NativeMojo.java

```diff
@@ -360,8 +389,9 @@ public FileVisitResult postVisitDirectory(Path source, IOException ioe) throws I
                     }
                     foundBundler = true;
 
-                    if (b.validate(params)) {
-                        b.execute(params, nativeOutputDir);
+                    Map<String, ? super Object> paramsToBundleWith = new HashMap<>(params);
```

Comment:
Some bundlers are messing with the map, so we are using their very own version of our map ;) (I suspected some bundler-settings leaking to cause this issue, so this also could be inside another PR)